src/pages/PositionsList/index.tsx

```diff
@@ -297,6 +319,124 @@ export const PositionsList = () => {
+          const { conditions } = row
+          const conditionId = conditions[0]?.conditionId ?? ''
+          if (conditions.length === 1) {
+            return <Hash href={`/conditions/${conditionId}`} value={conditionId} />
+          } else {
+            return (
+              <>
+                <Hash href={`/conditions/${conditionId}`} value={conditionId} />
+                <MoreLink
+                  onClick={() => {
+                    const hashes: HashArray[] = conditions.map(
+                      (condition: ConditionInformation) => {
+                        return {
+                          hash: condition.conditionId,
+                        }
+                      }
+                    )
+                    setOpenDisplayHashesTableModal(true)
+                    setHashesTableModal(hashes)
+                    setTitleTableModal('Condition Id')
+                    setTitleModal('Conditions')
+                    setUrlTableModal('conditions')
+                  }}
+                >
+                  (More...)
+                </MoreLink>
+              </>
+            )
+          }
+        },
+        name: 'Condition Id',
+        sortable: false,
+        minWidth: '350px',
+      },
+      {
+        // eslint-disable-next-line react/display-name
+        cell: (row: PositionWithUserBalanceWithDecimals) => {
+          const { conditions } = row
+          const oracle = conditions[0]?.oracle ?? ''
+          if (conditions.length === 1) {
+            return <Hash value={oracle} />
+          } else {
+            return (
+              <>
+                <Hash value={oracle} />
+                <MoreLink
+                  onClick={() => {
+                    const hashes: HashArray[] = conditions.map(
+                      (condition: ConditionInformation) => {
+                        return {
+                          hash: condition.oracle,
+                        }
+                      }
+                    )
+                    setOpenDisplayHashesTableModal(true)
+                    setHashesTableModal(hashes)
+                    setTitleTableModal('Oracle Id')
+                    setTitleModal('Oracles')
+                    setUrlTableModal('')
+                  }}
+                >
+                  (More...)
+                </MoreLink>
+              </>
+            )
+          }
+        },
+        name: 'Oracle',
+        sortable: false,
+        minWidth: '350px',
+      },
+      {
+        // eslint-disable-next-line react/display-name
+        cell: (row: PositionWithUserBalanceWithDecimals) => {
+          const { conditions } = row
+          const questionId = conditions[0]?.questionId ?? ''
+          if (conditions.length === 1) {
+            return <Hash value={questionId} />
+          } else {
+            return (
+              <>
+                <Hash value={questionId} />
+                <MoreLink
```

Comment:
We display a more link button, just in case exist more than one condition, should exist more than one oracle. I don't like this, maybe exist a better way to do this (same for question Id)